RecoLocalCalo/HcalRecAlgos/src/PulseShapeFitOOTPileupCorrection.cc

```diff
@@ -344,15 +346,7 @@ void PulseShapeFitOOTPileupCorrection::apply(const CaloSamples & cs,
    std::vector<float> fitParsVec;
 
    if(tstrig >= ts4Min_&& tsTOTen > 0.) { //Two sigma from 0
-     pulseShapeFit(energyArr, pedenArr, chargeArr, pedArr, gainArr, tsTOTen, fitParsVec, noiseADCArr);
-   }
-   else if((tstrig < ts4Min_||tsTOTen < 0.)&&(ts4Min_==0)){
-     fitParsVec.clear();
-     fitParsVec.push_back(0.);
-     fitParsVec.push_back(0.);
-     fitParsVec.push_back(0.);
-     fitParsVec.push_back(999.);
-     fitParsVec.push_back(false);
```

Comment:
it looks like removal of this ```else if ``` is actually triggering some small performance changes.
E.g. in 136.731 workflow about 10% of the hits were in this branch; so, their time and chi2 values have changed